demoapp/demo_app_functions.py

```diff
@@ -450,4 +512,642 @@ def update_csv(bill_num: str, title: str, summarized_bill: str, category: str, t
+
+    Returns: 
+        str: Query that includes task, context and instructions on summary generation
+
+    """
+
     
+    if llm_call_type == 'large':
+        query = f""" 
+                    Can you please explain what the following MA bill means to a regular resident without specialized knowledge? \n
+                    Please provide a one paragraph summary in a maximum of 4 sentences. Please be simple, direct and concise for the busy reader. \n
+                    Please use politically neutral language, keeping in mind that readers may have various ideological perspectives. \n
+                    Make bullet points if possible. 
+
+                    Note that the bill refers to specific existing chapters and sections of the Mass General Laws (MGL). Use the corresponding names of Mass General Law chapter and sections for constructing your summary.\n
+
+                    The bill title is: {getattr(bill_details, "bill_title")}\n
+
+                    The bill text is: {getattr(bill_details, "bill_text")}\n  
+
+                    The relevant section names are: {getattr(bill_details, "mgl_names")}\n
+                    
+                    The relevant committee information if available: {getattr(bill_details, "committee_info")}\n
+
+                    INSTRUCTIONS: 
+
+                    1. Only provide Summary, no other details are required. \n
+                    2. Do not provide tags or other extraneous text besides the summary. \n    
+                    3. Do not cite the title of the bill - the reader will already know that \n
+                    4. Do not cite specific section, chapter or title numbers of the MGL - the reader will not know what those sections are. \n
+                    5. Do not reference that this is a “MA” or “Massachusetts” bill - the reader will already know that. \n
+                    6. If referencing dates or other provisions of the bill, say that "this would happen if the bill is passed" rather than "this will happen". \n
+
+                    RESPONSE FORMAT:\n\n                Summary: [SUMMARY]
+                    """    
+
+    else: 
+        query = """
+                Can you please explain what the following MA bill means to a regular resident without specialized knowledge? \n
+                Please provide a one paragraph summary in a maximum of 4 sentences. Please be simple, direct and concise for the busy reader. \n
+                Please use politically neutral language, keeping in mind that readers may have various ideological perspectives. \n
+                Make bullet points if possible. 
+
+                Note that the bill refers to specific existing chapters and sections of the Mass General Laws (MGL). Use the corresponding names of Mass General Law chapter and sections for constructing your summary.\n
+
+                The bill title is: \"{title}"\
+
+                The bill text is: \"{context}"\
```

Comment:
I thought we were encapsulating in backticks

Reply:
Made the necessary changes